wicket/src/cli/rack_update.rs

```diff
@@ -41,10 +47,22 @@ use super::command::CommandOutput;
 pub(crate) enum RackUpdateArgs {
     /// Start one or more updates.
     Start(StartRackUpdateArgs),
+
     /// Attach to one or more running updates.
     Attach(AttachArgs),
+
     /// Clear updates.
     Clear(ClearArgs),
+
+    /// Dump artifacts and event reports from wicketd.
+    ///
+    /// Debug-only, intended for development.
```

Comment:
Nice :)

Do you have any concerns about this being available in production?

Reply:
No not really. There's nothing here that isn't in the UI, and this is clearly marked as a "debug" interface.